kyaml/resid/resid_test.go

```diff
@@ -93,61 +80,48 @@ var gvknStringTests = []struct {
 			Gvk:       Gvk{Group: "g", Version: "v", Kind: "k"},
 			Name:      "nm",
 		},
-		"g_v_k|nm",
+		"k.v.g/nm",
 	},
 	{
 		ResId{
 			Namespace: "ns",
 			Gvk:       Gvk{Version: "v", Kind: "k"},
 			Name:      "nm",
 		},
-		"~G_v_k|nm",
+		"k.v.[noGrp]/nm",
 	},
 	{
 		ResId{
 			Namespace: "ns",
 			Gvk:       Gvk{Kind: "k"},
 			Name:      "nm",
 		},
-		"~G_~V_k|nm",
+		"k.[noVer].[noGrp]/nm",
 	},
 	{
 		ResId{
 			Namespace: "ns",
 			Gvk:       Gvk{},
 			Name:      "nm",
 		},
-		"~G_~V_~K|nm",
+		"[noKind].[noVer].[noGrp]/nm",
 	},
 	{
 		ResId{
 			Gvk:  Gvk{},
 			Name: "nm",
 		},
-		"~G_~V_~K|nm",
-	},
-	{
-		ResId{
-			Gvk:  Gvk{},
-			Name: "nm",
-		},
-		"~G_~V_~K|nm",
-	},
-	{
-		ResId{
-			Gvk: Gvk{},
-		},
-		"~G_~V_~K|",
+		"[noKind].[noVer].[noGrp]/nm",
 	},
 	{
 		ResId{
 			Gvk: Gvk{},
 		},
-		"~G_~V_~K|",
+		"[noKind].[noVer].[noGrp]/",
 	},
 	{
 		ResId{},
-		"~G_~V_~K|",
+		"[noKind].[noVer].[noGrp]/",
```

Comment:
Not new, but why doesn't this one use the name placeholder?

Reply:
The GvknString method has the following body:

```
// GvknString of ResId based on GVK and name
func (id ResId) GvknString() string {
	return id.Gvk.String() + separator + id.Name
}
```

Which looks kind of odd to me to begin with and I'm unsure why it exists... but also I took a look and we aren't actually using this anywhere, and I'm not sure where we would use this if ever instead of ResId.String() so I'm inclined to just remove it entirely.